### PR TITLE
Updated  the community section

### PR DESCRIPTION
**Pull request by vandana1499**
fixes #39 

Description: Updated the community section

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [ ] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
d1

### CODE REVIEW

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
Looks good to me, over to you @EmperorYP7

File: client/src/components/Community/Community.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   
+}
+
+
+.Society-cards .heading
+{
+  text-align: center;
+  color: #ffffff;
+  font-style: italic;
+  font-weight: 200;
```
Comment (EmperorYP7) [inline client/src/components/Community/Community.css:10]: You may have the same tab size for consistency throughout the code.

File: client/src/components/Community/Community.js [hunk 2]
```diff
@@ -1,11 +1,59 @@
...
+        <Carousel>
+          <Carousel.Item>
+            <Card style={{ width: '18rem' }}>
+              <Card.Body>
+                <Card.Title>Card Title</Card.Title>
+                <Card.Subtitle className="mb-2 text-muted">Card Subtitle</Card.Subtitle>
+                <Card.Text>
+                  Some quick example text to build on the card title and make up the bulk of
+                  the card's content.
+    </Card.Text>
```
Comment (EmperorYP7) [inline client/src/components/Community/Community.js]: Fix this indentation.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Also, kindly pull all the new changes made to `houseofgeeks:front-end`

File: client/src/components/Community/Community.css (hunk 1 above)
Comment (vandana1499) [reply client/src/components/Community/Community.css:10]: I have fixed

File: client/src/components/Community/Community.js (hunk 2 above)
Comment (vandana1499) [reply client/src/components/Community/Community.js]: Fixed

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
This has been here for quite a while now! @EmperorYP7  and @tusharjain0022  Review this so that I can merge!

**Comment by tusharjain0022** [comment]
Everything seems alright 👍, LGTM!!

**Comment by ankiiitraj** [comment]
Congrats on your first contribution! 🎉 
Tip: Participating in PR reviews is also a contribution.

**Comment by EmperorYP7** [comment]
Ops.. Missed this one. Anyways, nice work @vandana1499! 🤩

**Comment by vandana1499** [comment]
Thanks, @EmperorYP7 !

**Pull request merged**